TWTValidation/JSON Validator/AST Nodes/TWTJSONSchemaKeyValuePairASTNode.m

```diff
@@ -29,9 +29,19 @@
 
 @implementation TWTJSONSchemaKeyValuePairASTNode
 
+- (instancetype)initWithKey:(NSString *)key valueSchema:(TWTJSONSchemaASTNode *)value
+{
+    self = [super init];
```

Comment:
If this isn’t your new designated initializer, you need to invoke `[self init]`. If it is your new designated initializer, you need to override `init` to invoke this (with `nil` arguments?)